build/do.rq

```diff
@@ -287,7 +287,7 @@ fetch_eopa_caps if {
 		not eopa_caps_tree[p]
 
 		# construct the URL to fetch the content from
-		r := rq.template("https://raw.githubusercontent.com/StyraInc/enterprise-opa/main/capabilities/{{.tag}}.json", {"tag": t})
+		r := rq.template("https://raw.githubusercontent.com/StyraInc/enterprise-opa/main/capabilities/{{.tag}}.regal.json", {"tag": t})
```

Comment:
This will fail for old tasks, iirc

Reply:
It should not have, since the test to fetch an old tag is based on the file on disk which is renamed to the format as before. However, sounds like we won't be needing now!